### ISSUE TITLE
three 123 remove getInverse() 

### ISSUE THREAD

**Issue by arpu** [gkjohnson/three-mesh-bvh#155]
Hey,

after update todays threejs 123 release i found
https://github.com/mrdoob/three.js/wiki/Migration-Guide#r122--r123

**Comment by gkjohnson** [comment]
Yes those lines need to be updated -- if you'd like to make a PR I'd be happy to merge it in!

As far as I know we need to change the following:

```js
matrixInv.getInverse( matrix );
```

to

```js
matrixInv.copy( matrix ).invert();
```

**Comment by arpu** [comment]
https://github.com/gkjohnson/three-mesh-bvh/pull/156

**Comment by gkjohnson** [comment]
Just published the fix in v0.3.0 to npm. Thanks again!
